app/stores/ada/TrezorConnectStore.js

```diff
@@ -114,6 +114,7 @@ export default class TrezorConnectStore
       TrezorConnect.manifest(trezorManifest);
 
       /** Preinitialization of TrezorConnect API will result in faster first response */
+      // $FlowFixMe we purposely don't want to wait for execution. Safe in practice.
```

Comment:
Pretty cool that now we have to specified that we want to ignore this on purpose :)

Reply:
To be honest it's not really good design that  we call an await inside setup but as I mentioned this should never cause  a problem in practice in store setup is called waaaay before it could ever be used by any real user. @sarveshwar-singh may have refactored this  whole  process in his work on WebAuthn so he may have to merge t his